common/src/main/java/tachyon/Constants.java

```diff
@@ -364,5 +364,10 @@
   public static final short DEFAULT_TFS_FULL_PERMISSION = (short) 0777;
   public static final short FILE_DIR_PERMISSION_DIFF = (short) 0111;
 
+  public static final String SECURITY_AUTHORIZATION_PERMISSION_ENABLED =
+      "tachyon.security.authorization.permission.enabled";
```

Comment:
These properties should also be document in tachyon/docs/Configuration-Settings.md